proposals/0243-character-operators.md

```diff
@@ -0,0 +1,159 @@
+# Character Literal Operators
+
+* Proposal: [SE-0243](0243-character-operators.md)
+* Authors: [Dianna ma (“Taylor Swift”)](https://github.com/kelvin13), [John Holdsworth](https://github.com/johnno1962)
+* Review manager: [Ben Cohen](https://github.com/airspeedswift)
+* Status: **Second review** 
+* Implementation: [apple/swift#71749](https://github.com/apple/swift/pull/71749)
```

Comment:
Since this proposal is a major substantive revision of a proposal that has undergone review, it should have a "Previous Revision" field that links to the previous revision.

```suggestion
* Implementation: [apple/swift#71749](https://github.com/apple/swift/pull/71749)
* Previous Revision: [1](https://github.com/apple/swift-evolution/blob/9713526f3423270c27082c620c75b2e5bc92050e/proposals/0243-codepoint-and-character-literals.md)
```